usr.sbin/mfiutil/mfi_bbu.c

```diff
@@ -50,10 +50,21 @@ mfi_autolearn_period(uint32_t period, char *buf, size_t sz)
 
 	tmp = buf;
 	if (d != 0) {
-		tmp += snprintf(buf, sz, "%u day%s", d, d == 1 ? "" : "s");
+		int fmt_len;
+		fmt_len = snprintf(buf, sz, "%u day%s", d, d == 1 ? "" : "s");
```

Comment:
The proposed change won't hurt but I don't think `snprintf` can return a negative number here, no?

Reply:
It returns the number of characters that would be written if there was an unlimited buffer.  It might be > sz.
However, it can return < 0 if there's an error in formatting (well, the man page just says "an error")